library/psa_crypto_driver_wrappers.c

```diff
@@ -44,6 +44,18 @@
 #endif /* PSA_CRYPTO_DRIVER_TEST */
 
 /* Repeat above block for each JSON-declared driver during autogeneration */
+
+#if defined(PSA_CRYPTO_DRIVER_CC3XX)
+#ifndef PSA_CRYPTO_DRIVER_PRESENT
+#define PSA_CRYPTO_DRIVER_PRESENT
+#endif
+#ifndef PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT
+#define PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT
+#endif
+/* Change the path to fit your machine */
+#include "<path>/trusted-firmware-m/lib/ext/cryptocell-312-runtime/codesafe/src/psa_driver_api/include/cc3xx.h"
```

Comment:
```suggestion
#include "cc3xx.h"
```

However, we need the following in the CMakeListst.txt under the PSA driver API folder in TF-M:

```CMake
target_include_directories(${CC312_LIB_TARGET}
        PUBLIC
        include
        .
)
```